WoodenUtilities/src/main/java/com/bedrocklegends/woodenutilities/setup/WoodenItems.java

```diff
@@ -27,13 +28,18 @@
     public static final RegistryObject<Item> WOODEN_PLATE = ITEMS
         .register("wooden_plate", () -> new Item(new Item.Properties().group(WoodenItemGroup.INSTANCE)));
 
+    //Universal Resin Bucket
+    public static final RegistryObject<Item> RESIN_BUCKET = ITEMS.register("resin_bucket",
```

Comment:
Move `"resin_bucket"` to `WoodenConstants`

Reply:
Done in 43c6f0b